src/ApplePayGateway.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+namespace Omnipay\Vindicia;
+
+/**
+ * Vindicia ApplePay Gateway
+ * This Apple Pay Gateway provides the functionality to authorize, complete authorize and capture
+ * Apple Pay payments for Apple Pay on the web.
+ * 
+ * // TODO: Add documentation about functionality for capture and completeAuthorize methods to capture
+ * // and authorize payments.
+ * 
+ * Apple Pay is available on all iOS devices with a Secure Element—an industry-standard, certified
```

Comment:
please fix the spacing in docs

Reply:
Is there a specific line that you are referring to? The spacing in this doc block is the same as PayPalPurchaseRequest: https://github.com/vimeo/omnipay-vindicia/blob/master/src/Message/PayPalPurchaseRequest.php

Reply:
just put a double dash between "Element" and "an" to make it easier to read

Reply:
done!